project/i-padiolleau/README.rst

```diff
@@ -0,0 +1,49 @@
+##Basic Information
+https://github.com/i-padiolleau/optimization-for-composite-type-data-structure
+
+##Problem to Solve
+When it comes to optimize a code that manipulate at signicant amount of data, it is important to consider how the data is access to adapt the data structure used in the code. 

```

Comment:
What will be the problem that manipulate significant amount of data in your project?

Reply:
I add specification on the problem I wnat to work on. Its about population data computation

Reply:
The tangible objective will help you focus.  It sounds good.